### ISSUE TITLE
User Story: **Create Model**

### ISSUE THREAD

**Issue by whelew** [whelew/P-Five-Mildew#8]
As a **Data Engineer** I can **train a model using a train set and a validation set** so that **the model can learn the relationships of the dataset**

**Acceptance Criteria:**

- Create a neural network model.

- Add basic hyperparameters
